integration-cli/docker_cli_stats_test.go

```diff
@@ -12,6 +12,7 @@ import (
 )
 
 func (s *DockerSuite) TestStatsNoStream(c *check.C) {
+	// Windows does not support stats
```

Comment:
Maybe we can rename this file from `docker_cli_stats_test.go` to `docker_cli_stats_unix_test.go` ? :wink: